lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -160,6 +160,10 @@ def clear_line
         print "\r" + ' ' * @max_length + "\r"
       end
 
+      def assumeyes(assume)
```

Comment:
Defining this as `def assumeyes=(value)` would feel more self-explanatory on what it does.

Reply:
I fixed this in the Hammer feature PR. It will get here with a rebase.

Reply:
hammer PR got merged, I guess we should update this now

Reply:
Oh I've added it to the predecessor, this needs to be removed